swift/apple/FirezoneKit/Sources/FirezoneKit/Models/WebAuthSession.swift

```diff
@@ -36,6 +36,11 @@ struct WebAuthSession {
         do {
           try await store.signIn(authResponse: authResponse)
         } catch {
+          let alert = NSAlert()
```

Comment:
I think this should just be a regular notification instead. I'd imagine that for most users, Firezone is just something that runs in the background and is basically never the reason why they booted their computer. Thus, grabbing their attention when it doesn't work seems wrong.

Reply:
This only happens when they actively try to sign in, so they are already present and waiting for a result. We can't surface errors otherwise because we can't guarantee the UI is running, which is needed to trigger the notifications from.

Reply:
So this isn't called for "auto-sign in" when we already have a token around and Firezone starts up?

Reply:
No this calls the same thing that does. We only alert upon the error here.

Reply:
This is specifically the web view sign in callback.